src/content/components/PriorityGuide/PriorityGuide.js

```diff
@@ -1,17 +1,29 @@
 import React from "react";
 import styles from "./PriorityGuide.scss";
+import { prefs } from "../../lib/prefs";
 
 export class PriorityGuide extends React.PureComponent {
   constructor(props) {
     super(props);
-    this.state = { priorityGuideCollapsed: false };
+    this.onPrefChange = this.onPrefChange.bind(this);
     this.onToggleCollapse = this.onToggleCollapse.bind(this);
```

Comment:
This method no longer references `this` so no need to bind it

Reply:
It references `this` for `this.setState({ priorityGuideCollapsed: newValue });`

Reply:
That's onPrefChange, not onToggleCollapse. But now that you've changed onToggleCollapse to use `this.state` it does need to be bound, so my comment isn't applicable anymore